src/deluge/gui/menu_item/osc/sync.h

```diff
@@ -25,7 +25,8 @@ class Sync final : public Toggle {
 	using Toggle::Toggle;
 	void readCurrentValue() override { this->setValue(soundEditor.currentSound->oscillatorSync); }
 	void writeCurrentValue() override { soundEditor.currentSound->oscillatorSync = this->getValue(); }
-	bool isRelevant(Sound* sound, int32_t whichThing) override {
+	bool isRelevant(ModControllableAudio* modControllable, int32_t whichThing) override {
+		Sound* sound = static_cast<Sound*>(modControllable);
```

Comment:
I don't think these are valid - the mod controllable might not be a sound now, so that needs to be checked

Reply:
Hm actually probably fine since you can't reach these leaf menus if you're not editing a sound